src/diffusers/commands/env.py

```diff
@@ -152,7 +152,7 @@ def run(self) -> dict:
 
         info = {
             "🤗 Diffusers version": version,
-            "Platform": f"{platform.freedesktop_os_release().get('PRETTY_NAME', None)} - {platform.platform()}",
+            "Platform": platform.platform(),
```

Comment:
Could we perhaps check the Python version first and realize the usage accordingly? That might be better no? 

https://github.com/huggingface/diffusers/blob/67b3fe0aaeddda19e8f61316a9b2fcceba4a4451/src/diffusers/utils/testing_utils.py#L387